lib/updateDeps.js

```diff
@@ -35,26 +35,35 @@ const getNextPreVersion = (pkg, tags) => {
 	const lastVersion = pkg._lastRelease && pkg._lastRelease.version;
 	const version = `1.0.0-${pkg._preRelease}.1`;
 
+	// Extract tags:
+	// 1. Set filter to extract only package tags
+	// 2. Get tags from a branch considering the filters established
```

Comment:
Lets add a note: `// TODO replace {cwd: '.'} with multiContext.cwd`. In order not to forget.

Reply:
@davikawasaki,

Or perhaps it's better to do it right here. `cwd` is already present in the context, so we can pass it down.
```
updateManifestDeps(pkg, cwd);
```